webpack/config.production.js

```diff
@@ -1,9 +1,6 @@
-'use strict';
-
-const
-    webpack = require('webpack'),
-    plugins = require('./plugins'),
-    layout = require('../layout');
+var webpack = require('webpack');
```

Comment:
vars usage violates our style-guide,
Please use const where possible, and let when mutation is neseccary